index.yaml

```diff
@@ -12,13 +12,13 @@ entries:
       catalog.cattle.io/release-name: fleet
     apiVersion: v2
     appVersion: 0.3.6-rc2
-    created: "2021-06-18T15:16:27.617227-07:00"
+    created: "2021-06-21T15:54:59.221518-07:00"
     dependencies:
     - condition: gitops.enabled
       name: gitjob
       repository: file://./charts/gitjob
     description: Fleet Manager - GitOps at Scale
-    digest: e0851254ef96f419c2d2bd0976ffe1651e85b57a3a3c895ae83c7b3df972e09f
+    digest: 5254dee03378d776bccdf95cc307801b98154bcbc14931b08a431fd347f0007b
```

Comment:
This is a known issue that @arvind and I are aware of and need to investigate. It does not affect the PR.